src/VisualStudio/Core/Def/ChangeSignature/ChangeSignatureDialogViewModel.ParameterViewModels.cs

```diff
@@ -93,6 +93,14 @@ public bool NeedsBottomBorder
         }
 
         public virtual string CallSiteValue => string.Empty;
+
+        public string RemovedParameterAutomationText
+        {
+            get
+            {
+                return $"{ServicesVSResources.Removed} {FullAutomationText}";
```

Comment:
This might be tricky for non-english locales. Maybe you could use `Removed {0}` as the localizable string format. Then the localization team could reorder things here if necessary.